cli.js

```diff
@@ -4,29 +4,95 @@
 /* eslint no-console:0 */
 
 const katex = require("./");
-let input = "";
+const fs = require('fs');
 
 // Skip the first two args, which are just "node" and "cli.js"
 const args = process.argv.slice(2);
 
 if (args.indexOf("--help") !== -1) {
     console.log(process.argv[0] + " " + process.argv[1] +
                 " [ --help ]" +
-                " [ --display-mode ]");
+                " [ --display-mode ]" +
+                " [ --throw-on-error ]" +
+                " [ --error-color color ]" +
+                " [ --max-size size ]" +
+                " [ --input inputFile ]" +
+                " [ --output outputFile ]"
+               );
 
     console.log("\n" +
                 "Options:");
-    console.log("  --help            Display this help message");
-    console.log("  --display-mode    Render in display mode (not inline mode)");
+    console.log("  --help              Display this help message");
+    console.log("  --display-mode      Render in display mode (not inline mode)");
+    console.log("  --no-throw-on-error Render unsupported commands as text");
+    console.log("  --error-color       Color unsupported commands are rendered in");
+    console.log("  --max-size          User-specified sizes are capped to maxSize");
```

Comment:
Should specify what units the cap is in.

Reply:
Same as above and I agree.